bundle/bin/rke2-uninstall.ps1

```diff
@@ -100,6 +100,7 @@ function Invoke-HNSRequest {
 
 # cleanup
 Write-Host "Beginning the uninstall process"
+Read-Host "Node must be drained and removed from the cluster before proceeding. Press Enter to continue."
```

Comment:
We don't require this for k3s or rke2 on linux. Is this just to give us less stuff to clean up when uninstalling?

Reply:
Based on my experience, if we don't do this, the script does not work correctly. The problem is that all resources of containerd need to be gone before stopping and uninstalling kubelet/containerd, otherwise, there are remaining files which are impossible to remove. If there are pods running in this node, it's impossible to clean everything because immediately after killing containerd tasks with this script, new tasks are recreated.

In linux, with the correct permissions (i.e. sudo), we can remove everything in `/var/lib/rancher`. However in Windows we can't if the files belong to a process that does not exist anymore (containerd). The workaround I found is rebooting the host so that Windows "forgets" about that process